lib/rspec/matchers.rb

```diff
@@ -769,8 +769,9 @@ def respond_to(*names)
     #
     # @example
     #   expect(5).to satisfy { |n| n > 3 }
-    def satisfy(&block)
-      BuiltIn::Satisfy.new(&block)
+    #   expect(5).to satisfy("be greater than 3") { |n| n > 3 }
```

Comment:
Do you mind adding a `# @param` declaration for the argument as well?  To see how it will render in our docs, use `bundle exec yard server --reload`.